Makefile

```diff
@@ -22,6 +22,8 @@ TEST_TYPE ?= wesql
 GIT_COMMIT  = $(shell git rev-list -1 HEAD)
 GIT_VERSION = $(shell git describe --always --abbrev=0 --tag)
 GENERATED_CLIENT_PKG = "pkg/client"
+GENERATED_DEEP_COPY_FILE = "zz_generated.deepcopy.go"
```

Comment:
suggest not to modify the unrelated files of the issue that this PR tries to resolve.